### ISSUE TITLE
CLI: --mute

### ISSUE THREAD

**Issue by telephone** [mako-framework/framework#131]
__Version 4.4__

As it stands, `--mute` is only half implemented. If a command requests user input, you're left with a blank line waiting for input.

If I create a question without a default, and does not allow an empty answer (while loop)... Well, you get the idea.

__Solutions:__

1. Create a `--force` option which will answer default to all answers where possible
    - If input does not have a default, throw an error
    - If `--mute` is enabled, apply `--force`
2. Have `--mute` answer default where possible, and throw error if input does not have a default
    - In other words, `--mute` does the same as option 1 but doesn't add a new global option
3. Add an option that disables `--mute` for the given command
    - A variable within `reactor\Command` -> `Command::$allowMute`
    - If `--mute` is enabled, throw an error

**Comment by freost** [comment]
My use case for the ```--mute``` option was to use it for commands that don't need any user interaction (e.g. cron jobs or queue workers). It's basically just the same as sending all output to ```/dev/null```.

I could add a ```Output::isMuted()``` method so that you can enable/disable output when needed.

**Comment by telephone** [comment]
> I could add a Output::isMuted() method so that you can enable/disable output when needed.

How about moving `--mute` to `cli\Application`. Right now it's the only hard-coded option within Reactor.
That way behaviour stay identical, and devs can implement their own `--mute` if they wish.

```php
        $reactor->registerCustomOption('mute', 'Disables all output', function($output)
        {
            $output->mute();
        });
```

**Comment by freost** [comment]
Yeah that would work too. I'll do both :)
